xwiki-platform-core/xwiki-platform-test/xwiki-platform-test-docker/src/main/java/org/xwiki/test/docker/junit5/AbstractContainerExecutor.java

```diff
@@ -51,4 +52,25 @@ protected void start(GenericContainer container, TestConfiguration testConfigura
         // Display logs after the container has been started so that we can see problems happening in the containers
         DockerTestUtils.followOutput(container, getClass());
     }
+
+    /**
+     * Utility method to mount a directory from the host to the container with some optimization for different OS.
+     * @param container the container on which to mount the directory.
+     * @param sourceDirectory the path of the directory on the host to mount in the container.
+     * @param targetDirectory the target where to mount the directory in the container.
+     * @since 10.11RC1
+     */
+    protected static void mountFromHostToContainer(GenericContainer container, String sourceDirectory,
```

Comment:
I don't like static too much (causes all sort of issues, including for testing when it sets some states).